lib/git-interface/comment.ts

```diff
@@ -0,0 +1,55 @@
+export class GitComment
+{
+    public owner: string;
+    public repo: string;
+    public issue_id: number;
```

Comment:
comments could also be on PRs and MRs, right? for github those IDs are unique, but maybe not for gitlab

Reply:
I just checked. GitLab's comments (called notes in the API) at least appear to be unique.